### ISSUE TITLE
Remove six dependency

### ISSUE THREAD

**Issue by dvzrv** [pyocd/pyOCD#1334]
The six dependency should not be needed anymore, as the project only supports python3 \o/

**Comment by flit** [comment]
Yup! I'm working on removing six, a little at a time. I think the only remaining uses are `ensure_bytes()` and `ensure_str()`.

**Comment by DavidLesnjak** [comment]
Fixed in [#1812](https://github.com/pyocd/pyOCD/pull/1812).  The last remaining uses of six have been removed, so pyOCD no longer depends on six.
